app/include/user_config.h

```diff
@@ -11,11 +11,13 @@
 // #define FLASH_16M
 #define FLASH_AUTOSIZE
 #define FLASH_SAFE_API
-// #define DEVELOP_VERSION
-#define FULL_VERSION_FOR_USER
 
-#define USE_OPTIMIZE_PRINTF
+// Byte 107 of esp_init_data_default, only one of these 3 can be picked
+#define ESP_INIT_DATA_ENABLE_READVDD33
+//#define ESP_INIT_DATA_ENABLE_READADC
```

Comment:
> I would like to build the firmware...what or how is the best root to take?

Not to ask such questions in a 2-months old commit comment would be a good start. Then you can familiarize yourself with http://nodemcu.readthedocs.org/en/dev/en/build/.